# Configuration of rrt planner
iterations: 100000
deltaX: 0.5
greedy: 0.1
laneWidth: 2.0
knn: 10
bias: 1.0
K: 3
